.travis/checks/check-signoff.py

```diff
@@ -15,11 +16,17 @@
 def isAutomaticMerge(sha):
     """
     Check whether a commit is a non-conflicting merge. Such merge
-    doesn't have any changes in it
+    either doesn't have any changes in it or has a commit message
+    matching a certain regular expression
     """
     cmd = ['git', 'show', '--format=', '--raw', sha]
     commit = subprocess.check_output(cmd).decode("utf-8")
-    return len(commit) == 0
+    if len(commit) == 0:
+        return True
+
+    cmd = ['git', 'log', '-n1', '--format=%B', sha]
+    commit_message = subprocess.check_output(cmd).decode("utf-8")
+    return re.match('Merge [0-9a-f]{7} into [0-9a-f]{7}\n\n', commit_message)
```

Comment:
Avoid being strict on the SHA size in the commit message, as git might change this (either related to SHA256 or what the significant SHA length is). Suggest just using '+'.